bootstrap/spigot/build.gradle.kts

```diff
@@ -1,4 +1,4 @@
-val paperVersion = "1.17.1-R0.1-SNAPSHOT" // Needed because we do not support Java 17 yet
+val paperVersion = "1.19-R0.1-SNAPSHOT" // Needed because we do not support Java 17 yet
```

Comment:
Remove the comment here since it's outdated.

Reply:
https://github.com/GeyserMC/Geyser/pull/3131/commits/2dbd39c5a437ee14c589acb1ab2576c9779725b9